docs/userGuide/syntax/dates.mbdf

```diff
@@ -0,0 +1,105 @@
+## Dates
+
+{% macro njcode(raw) %}<code>{<a/>{ {{ raw }} }}</code>{% endmacro %}
+
+**Markbind supports date formatting and simple calculations** as a Nunjucks filter.
+
+**Syntax:** {{ njcode('baseDate | date(format, daysToAdd)') }}
+
+<span id="main-example">
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+<box><span>
+{{ njcode('set base1 = "2019-08-12"') }}<br/>
+{{ njcode('set format1 = "DD MM YYYY"') }}<br/>
+{{ njcode('set format2 = "ddd DD/MM"') }}<br/>
+{{ njcode('base1 | date') }} // Mon 12 Aug<br/>
+{{ njcode('base1 | date(format1)') }} // 12 08 2019<br/>
+{{ njcode('base1 | date(format1, 10)') }} // 22 08 2019<br/>
+{{ njcode('base1 | date(format2, 10)') }} // Thu 22/08<br/>
+</span></box>
+
+</span>
+
+<span id="output">
+Mon 12 Aug
+
+12 08 2019
+
+22 08 2019
+
+Thu 22/08
+</span>
+</include>
+
+<box type="info">
+The base date follows the format: `YYYY-MM-DD`
+
+It can be supplied using [page variables](../reusingContents.html#variables) for convenience.
+</box>
+
+
+### Formatting
+
+The date can be formatted to suit your needs
+
+Default format: `ddd D MMM` e.g. Fri 6 Mar
+
+**Brief reference**
+
+Token | Output
+- | -
+D | 1
+Do | 1st
+DD | 01
+M | 1
+MM | 01
+MMM | Jan
+MMMM | January
+YY | 19
+YYYY | 2019
+
+Full formatting reference available [here](https://momentjs.com/docs/#/displaying/format/).
+
+Example
+<span id="examples" class="d-none">
+<include boilerplate src="outputBox.md">
+<span id="code">
+
+{{ njcode('set base1 = "2019-08-12"') }}
+
+{{ njcode('set format1 = "DD MM YYYY"') }}
+
+{{ njcode('set format2 = "ddd DD/MM"') }}
+
+{{ njcode('base1 | date') }} // Mon 12 Aug
+
+{{ njcode('base1 | date(format1)') }} // 12 08 2019
+
+{{ njcode('base1 | date(format1, 10)') }} // 22 08 2019
+
+{{ njcode('base1 | date(format2, 10)') }} // Thu 22/08
```

Comment:
If the base is defined in a different page, it needs to be imported first as per normal nunjucks practice?
Also, if this `set` is different from the normal nunjucks `set` (`{% set format1 = "DD MM YYYY" %}`), may be use a different keyword to avoid confusion?

Reply:
It's the normal nunjucks `set`. The base could also be defined in the global `variables.md` or in another page and imported normally as a nunjucks variable. The syntax above is a macro I created to format nunjucks code in Markbind pages.

Reply:
What I mean is, the syntax you give here (as shown in the generated UG) is _almost_ the same as the nunjucks syntax to set variables, not not exactly the same.
normal: `{% set format1 = "DD MM YYYY" %}`
vs: `{{ set format1 = "DD MM YYYY" }}`

![image](https://user-images.githubusercontent.com/1673303/76698017-e5a53500-66d8-11ea-87d9-76b37f6e8d3e.png)

Are they equivalent? Or just a mistake in your documentation?

Reply:
Oh my bad that was a mistake